phpBB/adm/style/tooltip.js

```diff
@@ -164,52 +170,49 @@ function create_element(tag, c) {
 * Correct positioning of tooltip container
 */
 function locate(e) {
```

Comment:
Sure, just didn't modify it for now. Still slightly working on the overall behavior.